onnxruntime/core/providers/cuda/cuda_execution_provider.h

```diff
@@ -126,6 +125,8 @@ class CUDAExecutionProvider : public IExecutionProvider {
 
   bool use_ep_level_unified_stream_ = false;
 
+  cuda::tunable::CudaTuningContext tuning_context_;
```

Comment:
> `cuda::tunable::CudaTuningContext`

Declare as `mutable cuda::tunable::CudaTuningContext` to document that it can be changed via a pointer retrieved by a call to `GetTuningContext`.